closure-deps/bin/closuremakedeps.js

```diff
@@ -129,11 +140,11 @@ function promisify(fn, ...args) {
 
 /**
  * @param {string} pathToScan
- * @param {!Set<string>} excluded
+ * @param {!Array<string>} excluded
```

Comment:
Nit: maybe name this excludedGlobs so it's clearer these are not file path references.